pep_sphinx_extensions/tests/test_pep_lint.py

```diff
@@ -0,0 +1,53 @@
+import importlib.util
+import sys
+from pathlib import Path
+
+# Import "pep-lint.py" as "pep_lint"
+PEP_LINT_PATH = Path(__file__).resolve().parent.parent.parent.joinpath("pep-lint.py")
+spec = importlib.util.spec_from_file_location("pep_lint", PEP_LINT_PATH)
+sys.modules["pep_lint"] = pep_lint = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(pep_lint)
```

Comment:
Entirely as I'd expect to execute a script with a hypen rather than an underscore (I frequently have this problem with cherry-picker vs cherry_picker!).